src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -404,22 +404,49 @@ subroutine CanopyFluxes(bounds,  num_exposedvegp, filter_exposedvegp,
     integer  :: iv
     logical  :: is_end_day                               ! is end of current day
 
+    real(r8) :: dbh(bounds%begp:bounds%endp)       !diameter at breast height of vegetation
+    real(r8) :: cp_leaf(bounds%begp:bounds%endp)   !heat capacity of leaves
+    real(r8) :: cp_stem(bounds%begp:bounds%endp)   !heat capacity of stems
+    real(r8) :: rstem(bounds%begp:bounds%endp)     !stem resistance to heat transfer
+    real(r8) :: dt_stem(bounds%begp:bounds%endp)   !change in stem temperature
+    real(r8) :: frac_rad_abs_by_stem(bounds%begp:bounds%endp)     !fraction of incoming radiation absorbed by stems
+    real(r8) :: lw_stem(bounds%begp:bounds%endp)   !internal longwave stem
+    real(r8) :: lw_leaf(bounds%begp:bounds%endp)   !internal longwave leaf
+    real(r8) :: sa_stem(bounds%begp:bounds%endp)   !surface area stem m2/m2_ground
+    real(r8) :: sa_leaf(bounds%begp:bounds%endp)   !surface area leaf m2/m2_ground
+    real(r8) :: sa_internal(bounds%begp:bounds%endp)   !min(sa_stem,sa_leaf)
+    real(r8) :: uuc(bounds%begp:bounds%endp)       ! undercanopy windspeed
+    real(r8) :: carea_stem                         !cross-sectional area of stem
+
+    ! biomass parameters
+    real(r8), parameter :: k_vert = 0.1            !vertical distribution of stem
```

Comment:
Shouldn't these all go on the params file?

Reply:
@swensosc can you comment on these new parameters in CanopyFluxesMod -- are they something that would be adjusted and hence should be on the params file?

Reply:
We decided to just add a comment about this. They don't need to go on the params file right now.